in_toto/models/metadata.py

```diff
@@ -146,13 +146,36 @@ class Envelope(SSlibEnvelope, Metadata):
 
     @classmethod
     def from_signable(cls, signable: Signable) -> "Envelope":
-        """Creates DSSE envelope with signable bytes as payload."""
+        """Creates DSSE envelope with signable bytes as payload.
 
+        Note:
+        This method intentionally bypasses the `signable_bytes` property of
+        the `Signable` class to avoid canonicalization, which is contrary
+        to DSSE's design principles. DSSE aims to avoid the complexities
+        and potential issues of canonicalization by using a straightforward
+        JSON serialization approach.
+
+        While this method currently expects an instance of `Signable` for
+        compatibility reasons, it is the non-canonical serialization that
+        is crucial for DSSE envelopes. The payloads do not strictly need
+        to subclass `Signable`, but they must be serializable to JSON.
```

Comment:
This is great, but maybe a bit wordy. Would you mind condensing it a little? The key points we want to convey are:

- The method takes a `Signable`, only because it is a base class for `Link` and `Layout`.
- `Signable.signable_bytes` is ignored, because canonical json is is undesired in the DSSE context.
-  Instead, this method serialises the `Signable` using regular JSON.